assets/javascript/components.js

```diff
@@ -49,8 +49,18 @@ registrationCard.innerHTML = `Matrícula: ${user[0]['infos'][0]['matricula']}`
 let emailCard = document.querySelector(".emailCard")
```

Comment:
Da linha 45 a 50, pode excluir essa parte do código. Está se repetindo no for of que eu criei.